lib/java/com/google/android/material/lists/MaterialListAdapter.java

```diff
@@ -0,0 +1,22 @@
+package com.google.android.material.lists;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+import androidx.recyclerview.widget.RecyclerView.ViewHolder;
+
+/** The parent adapter that defines behavior for specific adapters to define */
+public abstract class MaterialListAdapter extends RecyclerView.Adapter<ViewHolder> {
```

Comment:
I know the work on `MaterialListAdapter` is scheduled for later, but will `androidx.recyclerview.widget.ListAdapter` and `androidx.paging.PagedListAdapter` fit into this somehow? You don't need to answer this elaborate. A yes or no will suffice.

Reply:
We may support paged data types later on, but the answer is no. If you want to customize the data source/ adapter you currently have two options:
- Customize how you use MaterialListBinder
- Use the ViewHolders in your own adapters